### ISSUE TITLE
problem sync

### ISSUE THREAD

**Issue by mj-nz** [jordan-thoms/2Degrees-Toolbox#6]
Hi, can't get the ap to sync.  login and password are correct as I can sign online but ap tells me wrong details on phone.

**Comment by jordan-thoms** [comment]
What version do you have?

**Comment by mj-nz** [comment]
1.5.8

Date: Mon, 11 Mar 2013 17:23:42 -0700

**Comment by jordan-thoms** [comment]
Yeah, it's actually not working for me either. (I don't live in NZ anymore so I don't notice when it breaks). I'll take a look tonight and see if It's a quick fix, I haven't got much time to work on the app anymore unfortunately.

**Comment by mj-nz** [comment]
Sweet as and thanks very much

Date: Mon, 11 Mar 2013 17:29:09 -0700
